compiler/src/iree/compiler/Codegen/Common/TileSizeSelection.cpp

```diff
@@ -113,6 +113,20 @@ SmallVector<int64_t> TilingConfig::getFusableLevels() {
   }
 }
 
+SmallVector<int64_t> TilingConfig::getCacheLevels() {
+  switch (getNumTilingLevels()) {
+  case 0:
```

Comment:
For later, but I think it would be better to attach more semantic meaning to the values using enums.